xlml/apis/task.py

```diff
@@ -308,16 +308,16 @@ def clean_up(self, queued_resource: airflow.XComArg) -> DAGNode:
 
 
 @dataclasses.dataclass
-class TpuXpkTask(BaseTask):
-  """This is a class to set up tasks for TPU provisioned by XPK tool.
+class XpkTask(BaseTask):
+  """This is a class to set up tasks for TPU/GPU provisioned by XPK tool.
 
   Attributes:
-    task_test_config: Test configs to run on this TPU.
-    task_gcp_config: Runtime TPU creation parameters.
+    task_test_config: Test configs to run on this TPU/GPU.
+    task_gcp_config: Runtime TPU/GPU creation parameters.
     task_metric_config: Metric configs to process metrics.
   """
 
-  task_test_config: test_config.TestConfig[test_config.Tpu]
+  task_test_config: test_config.Accelerator
```

Comment:
This type annotation is wrong. We don't have a type checker enabled right now, so I'm just eyeballing this. The only two test types that break here are `TpuGkeTest` and `GpuGkeTest`, right? In that case, the annotation would be `Union[TpuGkeTest, GpuGkeTest]`.

Reply:
Thank you Will, just to be clear, I should change this line to `task_test_config: Union[test_config.TpuGkeTest, test_config.GpuXpkTest]` right?

Reply:
Yeah